dist/origin-web-common-services.js

```diff
@@ -3755,6 +3755,8 @@ angular.module('openshiftCommonServices')
       deploymentconfigs:                {group: 'apps.openshift.io',          version: 'v1',      resource: 'deploymentconfigs' },
 	    'deploymentconfigs/instantiate':  {group: 'apps.openshift.io',          version: 'v1',      resource: 'deploymentconfigs/instantiate' },
```

Comment:
nit: mind fixing the indentation here?

Reply:
in the `dist`, weird.  `src` is good, lemme rebuild & see whats up.